src/content/operations/deployment/helm/_index.en.md

```diff
@@ -74,16 +73,17 @@ helm install submariner-operator submariner-latest/submariner-operator \
         --set broker.token="${SUBMARINER_BROKER_TOKEN}" \
         --set broker.namespace="${BROKER_NS}" \
         --set broker.ca="${SUBMARINER_BROKER_CA}" \
+        --set broker.globalnet="${GLOBALNET}" \
+        --set submariner.serviceDiscovery=true \
         --set submariner.cableDriver=libreswan \ # or wireguard or vxlan
         --set submariner.clusterId="${CLUSTER_ID}" \
         --set submariner.clusterCidr="${CLUSTER_CIDR}" \
         --set submariner.serviceCidr="${SERVICE_CIDR}" \
         --set submariner.globalCidr="${GLOBAL_CIDR}" \
+        --set submariner.natEnabled="true" \ # disable this if no NAT will happen between gateways
```

Comment:
I know you're just carrying this over from the existing docs (so non-blocking feedback for this PR), but I noticed that our recommendation for `natEnabled="true"` here doesn't match our `natEnabled: false` default in https://github.com/submariner-io/submariner-charts/blob/devel/submariner-operator/values.yaml#L8 and I wonder which it should be? Should we change the config-default to `true` or the docs-default to `false`? @sridhargaddam @vthapar @aswinsuryan?

Reply:
Talking with this on the PR scrub today, the group-memory is that this is default-true in submariner-operator, so we should use that here.

Reply:
And we default to `true` in the parallel example for the bundle install method:

https://github.com/submariner-io/submariner-operator/blob/devel/bundle/manifests/submariner.clusterserviceversion.yaml#L49